components/dashboard/sections/collectives/common.tsx

```diff
@@ -31,37 +31,31 @@ export const cols: Record<string, ColumnDef<any, any>> = {
     accessorKey: 'collective',
     header: () => <FormattedMessage id="Collective" defaultMessage="Collective" />,
     cell: ({ row, table }) => {
-      const { intl, openCollectiveDetails } = table.options.meta as {
+      const { intl } = table.options.meta as {
         intl: IntlShape;
-        openCollectiveDetails: (c: HostedCollectiveFieldsFragment) => void;
       };
       const collective = row.original;
       const children = mapValues(groupBy(collective.childrenAccounts?.nodes, 'type'), 'length');
       const secondLine = toPairs(children)
         .map(([type, count]) => count && `${count} ${formatCollectiveType(intl, type, count)}`)
         .join(', ');
       return (
-        <button onClick={() => openCollectiveDetails?.(collective)} className="flex items-center">
+        <div className="flex items-center">
           <Avatar collective={collective} radius={48} className="mr-4" />
           <div className="flex flex-col items-start">
-            <div className="text-sm">{collective.name}</div>
-            <div>{secondLine}</div>
+            <div className="text-foreground">{collective.name}</div>
+            <div className="text-xs">{secondLine}</div>
           </div>
-        </button>
+        </div>
       );
     },
   },
   childCollective: {
     accessorKey: 'collective',
     header: () => <FormattedMessage id="Fields.name" defaultMessage="Name" />,
-    cell: ({ row, table }) => {
+    cell: ({ row }) => {
       const collective = row.original;
-      const { openCollectiveDetails } = table.options.meta as any;
-      return (
-        <button onClick={() => openCollectiveDetails?.(collective)} className="text-sm">
-          {collective.name}
-        </button>
-      );
+      return <div className="text-sm">{collective.name}</div>;
```

Comment:
Do we want to keep `text-sm` here?

Reply:
True, since we set the table class to text-xs in the collective details

Reply:
Added that